ide/static/js/data.js

```diff
@@ -522,18 +522,79 @@ export default {
         type: 'number',
         required: false
       },
-      weight_filler: {
+      dilation_h: {
+        name: 'Dilation height',
+        value: 1,
+        type: 'number',
+        required: false
+      },
+      dilation_w: {
+        name: 'Dilation width',
+        value: 1,
+        type: 'number',
+        required: false
+      },
+      weight_filler: { // Keras(kernel_initializer)
```

Comment:
Change this to "Maps to: kernel_initializer (Keras)"? Otherwise it might imply that this is a Keras only layer. Follow this convention everywhere.